pkg/events/logger_handler/logger_handler.go

```diff
@@ -0,0 +1,83 @@
+package logger_handler
```

Comment:
Also rename file to _handler.go_.

```suggestion
package logger
```